Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -402,6 +402,13 @@ header: <WebCore/AppKitControlSystemImage.h>
 enum class WebCore::AppKitControlSystemImageType : uint8_t {
     ScrollbarTrackCorner,
 };
+
+header: <WebCore/ScrollbarTrackCornerSystemImageMac.h>
+[AdditionalEncoder=StreamConnectionEncoder, RefCounted, ConstructSubclass=ScrollbarTrackCornerSystemImageMac] class WebCore::AppKitControlSystemImage {
+    [Validator='*m_controlType == WebCore::AppKitControlSystemImageType::ScrollbarTrackCorner'] WebCore::AppKitControlSystemImageType m_controlType;
```

Comment:
Let's remove this from the serialization if it can only have one valid value and it must be a certain value.  Then we can remove the RELEASE_ASSERT too

Reply:
Okay, will make that change next.  (Added `watchlist` support before I saw this.)